CMakeLists.txt

```
@@ -367,11 +354,11 @@ endif()
 
 # ZIP only contains source datastreams and kickstarts, people who
 # want sources to build from should get the tarball instead.
-if(SSG_TARGET_SCAP_VERSION VERSION_EQUAL "1.3")
+if(SSG_TARGET_OVAL_VERSION VERSION_EQUAL "5.11")
     ssg_build_zipfile("scap-security-guide-${SSG_VERSION}")
     ssg_build_nist_zipfile("scap-security-guide-${SSG_VERSION}-SCAP-1.3")
 else()
-    ssg_build_zipfile("scap-security-guide-${SSG_VERSION}-SCAP-1.2")
+    ssg_build_zipfile("scap-security-guide-${SSG_VERSION}-oval-${SSG_TARGET_OVAL_VERSION}")
```

`${SSG_TARGET_OVAL_VERSION}` contains `dot` in the value (`5.11`). Release artifacts for OVAL 5.10 are in the form of `scap-security-guide-0.1.45-oval-510.zip`. I wonder if this can impact anyone somehow.

---

```suggestion
    ssg_build_zipfile("scap-security-guide-${SSG_VERSION}-oval-${SSG_TARGET_OVAL_MAJOR_VERSION}${SSG_TARGET_OVAL_MINOR_VERSION}")
```
Would this be acceptable?

---

I just reverted to state before #4689 
https://github.com/ComplianceAsCode/content/pull/4689/files#diff-af3b638bc2a3e6c650974192a53c7291L268

This is a nice catch, as up to version 0.1.40, the 5.10 zip was named suffix `oval-5.10.zip`.

Do you feel strongly about this change?

---

It seems that no harm was caused by using a different names. So I'm ok leaving as it is right now.